libdl only exists on Linux. Use CMAKE_DL_LIBS instead.

Submitted upstream: https://github.com/dirkvdb/ffmpegthumbnailer/pull/132
--- CMakeLists.txt.orig	2016-04-16 16:57:31 UTC
+++ CMakeLists.txt
@@ -96,9 +96,8 @@ FIND_LIBRARY(SWSCALE_LIBRARY
 
 IF (ENABLE_GIO)
     FIND_PATH(DL_INCLUDES dlfcn.h)
-    FIND_LIBRARY(DL_LIBRARY dl)
     LIST(APPEND FFMPEGTHUMBNAILER_INCLUDE_DIRS ${DL_INCLUDES})
-    LIST(APPEND FFMPEGTHUMBNAILER_LIBRARIES ${DL_LIBRARY})
+    LIST(APPEND FFMPEGTHUMBNAILER_LIBRARIES ${CMAKE_DL_LIBS})
 ENDIF ()
 
 LIST(APPEND FFMPEGTHUMBNAILER_INCLUDE_DIRS ${AVFORMAT_INCLUDE_DIR} ${AVCODEC_INCLUDE_DIR} ${AVUTIL_INCLUDE_DIR} ${SWSCALE_INCLUDE_DIR})
